prowler/__main__.py

```diff
@@ -169,6 +171,21 @@ def prowler():
     # Extract findings stats
     stats = extract_findings_statistics(findings)
 
+    if args.slack:
+        if "SLACK_API_TOKEN" in os.environ and "SLACK_CHANNEL_ID" in os.environ:
+            _ = send_slack_message(
+                os.environ["SLACK_API_TOKEN"],
+                os.environ["SLACK_CHANNEL_ID"],
+                stats,
+                provider,
+                audit_info,
+            )
+        else:
+            logger.critical(
```

Comment:
Is this a `critical` error? I think with an error should be enough since the execution just finished but we cannot send it.

Reply:
It is a critical error like the ones we have when the generation of the output files fail.